src/main/java/bio/terra/cloudevents/v1/CloudEventType.java

```diff
@@ -0,0 +1,60 @@
+package bio.terra.cloudevents.v1;
+
+/**
+ * For a enumeration of all Cloud Event types, please refer to
+ * https://github.com/googleapis/google-cloudevents/tree/master/jsonschema/google/events/cloud
+ */
+public enum CloudEventType {
```

Comment:
I don't think we need this enum at this time.

Reply:
Here's the reason I feel the enum is important, based on the published [CloudEvents - Version 1.0](https://github.com/cloudevents/spec/blob/v1.0/spec.md#overview). One of the key take away is

- Event Formats specify how to serialize a CloudEvent with certain encoding formats. Compliant CloudEvents implementations that support those encodings MUST adhere to the encoding rules specified in the respective event format. All implementations MUST support the JSON format.

The Cloud Event formats and type strings aren't very well documented except in the above spec. Unfortunately, I couldn't find an equivalent Java implementation of these cloud event type strings so implement our own. 

Right now, for streaming cloud function, we only need the object finalize event type. Other event types in `public enum CloudEventType` will come in handy when we build cloud functions for duplicate and error handlings.